src/main/takeDetection/takeDetection.ts

```diff
@@ -101,4 +110,4 @@ export function findTakes(words: Word[], threshold = THRESHOLD): InjectableTakeG
   return takeGroups;
 }
 
-export default { getSimilarityScore };
+export default getSimilarityScore;
```

Comment:
Wait why are we exporting `getSimilarityScore` from this file when we export it from `sentenceSimilarity.ts`?

Reply:
Yeah weird dunno how that happened, removed